packages/core/utils/sanitizeHTML.js

```diff
@@ -0,0 +1,39 @@
+const getEnv = require('./universalEnv');
+const { getValueFromConfig } = require('../config/irving/getValueFromConfig');
+const {
+  IRVING_EXECUTION_CONTEXT,
+  BABEL_ENV,
+} = getEnv();
+
+let sanitizeHTML;
+
+if (
+  ! IRVING_EXECUTION_CONTEXT ||
+  'development_server' === IRVING_EXECUTION_CONTEXT ||
+  'production_server' === IRVING_EXECUTION_CONTEXT ||
+  'test' == BABEL_ENV
+) {
+  const createDOMPurify = require('dompurify');
+  sanitizeHTML = createDOMPurify(window);
+} else {
+  sanitizeHTML = require('dompurify');
+}
+
+module.exports = sanitizeHTML;
+module.exports.config = getValueFromConfig(
+  'sanitizeHTML',
+  {
+    ADD_TAGS: [
+      'iframe', 'link',
+    ],
+    ADD_ATTR: [
+      'rel', 'href', 'type', 'target', 'nohref', 'xml:lang', 'currenttime',
+      'duration', 'char', 'charoff', 'hspace', 'longdesc', 'vspace', 'autofocus',
+      'dirname', 'form', 'formaction', 'formenctype', 'formmethod',
+      'formnovalidate', 'formtaget', 'rules', 'abbr', 'axis',
+    ],
```

Comment:
I think the following are included/not needed here: `rel`, `href`, `type`

A couple I think need to be added: `aria-*`, `data-*`, `buffered`, `intrinsicsize`

Also, might be nice to alphabetize this list for readability.

Reply:
Looks like we had `rel`, `href`, and `type` in here for `link` tags, so I'm gonna keep those in for now. afaik ARIA and data attributes are supported out of the box, and configured separately. Will add the other two, though.